src/routing/index.html

```diff
@@ -1,8 +1,8 @@
 ---
 title: Routing
 description: |
-  Content routing is the way to determine where to find a given CID on the network;
-  specifically, which network peers provide specific CIDs. 
+  Routing is the way to determine where to find a given CID, Peer, or IPNS Record on the network;
```

Comment:
```suggestion
  Routing is the way to determine where to find a given resource (e.g. Data, Peer, or IPNS record).
```

The above might not be quite right but my two notes here are:
1. It might be better to define the category more broadly and give examples. Although also fair to just describe exactly what's in there now and broaden as we add more
2. Technically we don't look for CIDs, we have CIDs and look for some of the data they refer to (e.g. the block itself, the full DAG underneath it, etc.)

Not blockers though

Reply:
Makes sense, I've applied a shortened version of your suggestion.